nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-processors/src/main/java/org/apache/nifi/processors/mongodb/GetMongo.java

```diff
@@ -120,12 +123,30 @@ public ValidationResult validate(final String subject, final String value, final
         .addValidator(StandardValidators.POSITIVE_INTEGER_VALIDATOR)
         .build();
 
+    static final String JSON_TYPE_EXTENDED = "Extended";
+    static final String JSON_TYPE_NORMAL   = "Normal";
+    static final AllowableValue JSON_EXTENDED = new AllowableValue(JSON_TYPE_EXTENDED, "Extended JSON",
```

Comment:
Perhaps replace the developer-specific doc (toJson() from the Java driver, e.g.) with user-level info about [MongoDB Extended JSON](https://docs.mongodb.com/manual/reference/mongodb-extended-json/)?  This is a suggestion not a requirement :)